docs/concepts/managed-resources.md

```diff
@@ -9,42 +9,202 @@ indent: true
 
 ## Overview
 
-Managed resources are the Crossplane representation of the cloud
-[provider][provider] resources and they are considered primitive low level
-custom resources that can be used directly to provision external cloud resources
-for an application or as part of an infrastructure composition.
-
-For example, `RDSInstance` in AWS Provider corresponds to an actual RDS Instance
-in AWS. There is a one-to-one relationship and the changes on managed resources
-are reflected directly on the corresponding resource in the provider.
-
-You can browse [API Reference][api-reference] to discover all available managed
-resources.
+A Managed Resource is Crossplane's representation of a resource in an external
```

Comment:
I was on the fence about this but held off because historically we rarely abbreviate Managed Resource to MR. I don't see why we wouldn't though - I updated this paragraph to introduce the abbreviation but have not refactored to use it elsewhere. Most of `managed-resources-md` is just two documents I merged together, so I would prefer to avoid too much diff on the actual content in this PR.